src/java.base/share/classes/java/lang/Long.java

```diff
@@ -1138,7 +1138,7 @@ public static Long valueOf(String s, int radix) throws NumberFormatException {
      * interpreted as representing a signed decimal {@code long},
      * exactly as if the argument were given to the {@link
      * #parseLong(java.lang.String)} method. The result is a
-     * {@code Long} object that represents the integer value
+     * {@code Long} object that represents the {@code Long} value
```

Comment:
I think it should be `{@code long}`, i.e., the primitive type, not the class, for consistency with other numeric classes.